contrib/download-frozen-image-v2.sh

```diff
@@ -47,6 +47,37 @@ registryBase='https://registry-1.docker.io'
 authBase='https://auth.docker.io'
 authService='registry.docker.io'
 
+ARM_VARIANT=""
+
+fetch_arm_variant() {
+        if [ "$(go env GOHOSTOS)" = "linux" ];then
```

Comment:
Can you change this to use tabs for indentation?

Reply:
sorry~Fix it now😊